src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -977,7 +973,6 @@ so that we can apply CSS-filters to change the arrow color in themes */
 
 .search-results > a {
 	display: block;
-	width: 100%;
```

Comment:
There is slight display change with this one as the results seem less wide but the result looks good to me so all good.

Reply:
You can see it in the issue linked in the PR: https://github.com/rust-lang/rust/issues/85544